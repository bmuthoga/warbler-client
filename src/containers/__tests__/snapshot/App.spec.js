import React from 'react'
import toJson from 'enzyme-to-json'
import { shallow } from 'enzyme'
import App from '../../App'

describe('App Snapshot Tests', () => {
  it('should render the App component', () => {
    const wrapper = shallow(
      <App />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })  
})
