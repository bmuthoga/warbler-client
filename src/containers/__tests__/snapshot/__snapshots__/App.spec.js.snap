// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot Tests should render the App component 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <div
      className="onboarding"
    >
      <Connect(Navbar) />
      <withRouter(Connect(Main)) />
    </div>
  </BrowserRouter>
</Provider>
`;
